Tests/SentryTests/Protocol/SentryGeoTests.swift

```diff
@@ -0,0 +1,62 @@
+import XCTest
+
+class SentryGeoTests: XCTestCase {
```

Comment:
Beautiful tests, how did you come up with these 😄🙃 ?

Reply:
You could say i've got a taste....


... for Copy & Paste 😎